index.html

```diff
@@ -196,8 +196,26 @@ <h4>LET'S  CONNECT</h4>
             </ul>
             </div>
            </div>
-              </section> 
+              </section>
```

Comment:
[Optional] In order to make your webpage look better, it's important to follow the Figma template. So please try to remove the extra space from the right.

![image](https://user-images.githubusercontent.com/34779928/192622605-0333a190-33e0-4eba-abbf-f2d252bb3426.png)